File: Scripts/pageobjects/HomePage_po.py
from BasePage import BasePage


class HomePage(BasePage):
    signed = "//a[@class='nav-link ng-binding']"
    tv_new_article = "//a[@href='#!/editor/']"

    def signup_check(self):
        return self._driver.find_elements_by_xpath(self.signed)[0].text

    def launch_new_article(self):
        self._driver.find_elements_by_xpath(self.tv_new_article)[0].click()

File: angularjs-realworld-example-app-master/tests/PageObjects/HomePage.po.js
var HomePage = function(){
    this.title = 'Home — Conduit';
    this.signed = element.all(by.xpath("//a[@class='nav-link ng-binding']")).first();
    this.new_article = element.all(by.xpath("//a[@href='#!/editor/']")).first()
};

HomePage.prototype.signup_check = function() {
    return this.signed.text;
};

HomePage.prototype.launch_new_article = function() {
    return this.new_article.click();
};

module.exports = HomePage;